### PR TITLE
Use structopt for much nicer command line arguments parsing

### PR DESCRIPTION
**Pull request by mlatief**
- Use `structopt` for much nicer command line arguments parsing.
- Also for conversions utils use `mmmParser::parse` instead of `mmmBuilder::new`.
- And use `anyhow` for nicer error reporting.

### CODE REVIEW

**Comment by mlatief** [comment]
I will rebase before merge

**Review by drodil — APPROVED** [review]
Very nice refactoring but huge PR. Looks good to me!

**Comment by mlatief** [comment]
I might as well include and refactor the new calculators once they are merged.

File: rustybeer-util/src/conversions.rs [hunk 1]
```diff
@@ -1,299 +1,319 @@
...
-                    "m3" => Volume::from_cubic_meters(float_val.parse::<f64>()?),
-                    "μl" => Volume::from_drops(float_val.parse::<f64>()?),
-                    "dr" => Volume::from_drams(float_val.parse::<f64>()?),
-                    "l" => Volume::from_litres(float_val.parse::<f64>()?),
-                    "p" => Volume::from_pints(float_val.parse::<f64>()?),
-                    "ʒ" => Volume::from_pints(float_val.parse::<f64>()?),
-                    _ => Volume::from_litres(val.parse::<f64>()?),
+                    "cm3" => {
+                        measurements::Volume::from_cubic_centimeters(float_val.parse::<f64>()?)
+                    }
```
Comment (ProgrammerJoe93) [inline rustybeer-util/src/conversions.rs:63]: Just wondering why cm3 is a little different to the others

**Review by ProgrammerJoe93 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**